README.md

```diff
@@ -462,3 +462,26 @@ If you want to use simplePubSub server as the means of communication for 3rd par
 7. When running FCS include env variables:
     - deviceMac: `<DEVICE_UNDER_TEST>`
     - pubSubUrl: `ws://<YOUR_IP>:8080`
+
+# Dynamic Content Validation
+
```

Comment:
this is not the documentation I was talking about. You need to add fcs.fetchDeviceDetails to this list - https://github.com/rdkcentral/firebolt-certification-suite?tab=readme-ov-file#request-overrides

Reply:
added new override function fetchDeviceDetails in requestOverrides